tools/cockpit.spec

```diff
@@ -148,7 +148,7 @@ make -j4 check
 
 %install
 make install DESTDIR=%{buildroot}
-make install-test-assets DESTDIR=%{buildroot}
+make install-tests DESTDIR=%{buildroot}
```

Comment:
This is the way it's usually done. In addition, the ```/usr/share/cockpit/playground``` stuff, which is also part of cockpit-tests, comes from ```make install```.